pkg/controlplane/apiserver/config.go

```diff
@@ -96,7 +96,7 @@ type Extra struct {
 	// ServiceAccountIssuerDiscovery
 	ServiceAccountIssuerURL  string
 	ServiceAccountJWKSURI    string
-	ServiceAccountPublicKeys []interface{}
+	ServiceAccountPublicKeys serviceaccount.PublicKeyGetter
```

Comment:
nit: should we rename `ServiceAccountPublicKeys` to `ServiceAccountPublicKeysGetter` ?

Reply:
sure